server/bundler/extensions-loader.js

```diff
@@ -0,0 +1,34 @@
+const existsSync = require( 'fs' ).existsSync;
+const path = require( 'path' );
+const camelCase = require( 'lodash' ).camelCase;
+
+function getReducers( extensions = {} ) {
+	return extensions
+		.filter( ( extension ) => existsSync( path.join( 'client', 'extensions', extension, 'state', 'reducer.js' ) ) )
```

Comment:
>Can we use async stuff like this in a webpack loader? I'm no expert here.

Ah yeah, you have to opt in to it, but it's fairly straight-forward:

https://webpack.js.org/api/loaders/#async-loader